addons/crm/wizard/crm_lead_lost.py

```diff
@@ -9,20 +9,25 @@ class CrmLeadLost(models.TransientModel):
     _name = 'crm.lead.lost'
     _description = 'Get Lost Reason'
 
+    lead_ids = fields.Many2many('crm.lead')
     lost_reason_id = fields.Many2one('crm.lost.reason', 'Lost Reason')
     lost_feedback = fields.Html(
         'Closing Note', sanitize=True
     )
 
+    def default_get(self, fields_list):
+        defaults = super().default_get(fields_list)
+        defaults.setdefault('lead_ids', self._context.get('active_ids', []))
```

Comment:
Ideally should check that "lead_ids" is in fields_list beforehand.

It's good to keep this for backward compatibility and has a fallback, but can we also change the other side of the action to set "default_lead_ids" instead of magically using the context?

Reply:
I just removed the default_get override since that should be handled in the base default_get with `default_{field}` in context